pages/qrScanner.tsx

```diff
@@ -196,4 +200,4 @@ const QrScannerPage = () => {
   );
 };
 
-export default QrScannerPage;
+export default isAdmin(QrScannerPage);
```

Comment:
Should be authPage(Admin, ["admin"]) if you agree with my statements in my previous comments